Atomic/syscontainers.py

```diff
@@ -669,6 +669,17 @@ def _canonicalize_location(self, destination):
             sysroot.load()
             osname = sysroot.get_booted_deployment().get_osname()
             destination = os.path.realpath(os.path.join("/ostree/deploy/", osname, os.path.relpath(destination, "/")))
+            # Test that the ostree location and /var show the same file.
+            # If they do not then we must return /var.
+            test_file = str(uuid.uuid4())
+            with open(os.path.join(destination, '.' + test_file), 'w') as f:
```

Comment:
destination might not exist when this function is called, so it would be safer to create the file as sibling of destination

Reply:
It think it should be as it's both a parameter to the function as well as set a few lines prior.